script.js

```diff
@@ -0,0 +1,38 @@
+$(document).ready(function() {
+	getQuote();
+    $('.trigger').on('click', function() {
+        getQuote();
+    });
+
+    var prefix = "https://cors-anywhere.herokuapp.com/";
+	var tweetLink = "https://twitter.com/intent/tweet?text=";
+	var quoteUrl = "https://quotesondesign.com/wp-json/posts?filter[orderby]=rand&filter[posts_per_page]=1";
+
+	function getQuote() {
+    	$.getJSON(prefix + quoteUrl, createTweet);
+    	$.ajaxSetup({ cache: false });
+	};
+
+	function createTweet(input) {
+    	var data = input[0];
```

Comment:
from where you know that input is an array and contain at least one item. before you use it, check that response is an array with at least one element